util/test/sub_test

```diff
@@ -1032,6 +1033,26 @@ for testname in testsrc:
             if do_not_test:
                 break
 
+        elif (suffix=='.suppressif' and (os.access(f, os.R_OK))):
+            testskipiffile=True
+            skiptest=runSkipIf(f)
+            try:
+                skipme=False
+                if skiptest.strip() != "False":
+                    skipme = skiptest.strip() == "True" or int(skiptest) == 1
+                if skipme:
+                    suppressfile = open('./'+test_filename+'.suppressif', 'r')
```

Comment:
In python, using a with context is better for opening files (it ensure the file is closed even if exceptions are thrown). E.g.

```python
with open('./'+test_filename+'.suppressif', 'r') as suppressfile:
    suppressline = suppressfile.readline().strip()
```

Also, semicolons are not needed :smiley: